airline-web/app/controllers/NegotiationUtil.scala

```diff
@@ -517,7 +518,7 @@ case class NegotiationInfo(fromAirportRequirements : List[NegotiationRequirement
 
 object NegotiationRequirementType extends Enumeration {
   type NegotiationRequirementType = Value
-  val FROM_COUNTRY_RELATIONSHIP, TO_COUNTRY_RELATIONSHIP, EXISTING_COMPETITION, NEW_LINK, UPDATE_LINK, INCREASE_CAPACITY, INCREASE_FREQUENCY, EXCESSIVE_FREQUENCY, LOW_LOAD_FACTOR, FOREIGN_AIRLINE, NON_GATEWAY, STAFF_CAP, BAD_MUTUAL_RELATIONSHIP, OTHER = Value
+  val FROM_COUNTRY_RELATIONSHIP, TO_COUNTRY_RELATIONSHIP, EXISTING_COMPETITION, NEW_LINK, UPDATE_LINK, INCREASE_CAPACITY, INCREASE_FREQUENCY, EXCESSIVE_FREQUENCY, LOW_LOAD_FACTOR, FOREIGN_AIRLINE, NON_GATEWAY, GATEWAY, STAFF_CAP, BAD_MUTUAL_RELATIONSHIP, OTHER = Value
```

Comment:
@bertauchekurtis looks to me like we added but didn't remove the old var, so likely this will cause a fatal undefined error

Reply:
It does not need to be removed. I believe this is just an enumeration of the possible values for what a NegotiationRequirementType can be. For example, "EXISTING_COMPETITION" is never used except for right there on line 520. Leaving it in allows us to add a NON_GATEWAY penalty later more easily if we so wish as it keeps that kind of negotiation requirement as a defined type.

Reply:
Ah yea that makes sense!